bcda/models/models.go

```diff
@@ -248,22 +250,21 @@ func (aco *ACO) GetBeneficiaries(includeSuppressed bool) ([]CCLFBeneficiary, err
 		return cclfBeneficiaries, fmt.Errorf("unable to find cclfFile")
 	}
 
-	var suppressedHICNs []string
 	if !includeSuppressed {
-		db.Raw(`SELECT DISTINCT s.hicn
+		db.Raw(`SELECT DISTINCT s.blue_button_id
 			FROM (
-				SELECT hicn, MAX(effective_date) max_date
-				FROM suppressions 
-				WHERE effective_date <= NOW() AND preference_indicator != '' 
-				GROUP BY hicn
+				SELECT blue_button_id, MAX(effective_date) max_date
+				FROM suppressions
+				WHERE effective_date <= NOW() AND preference_indicator != ''
+				GROUP BY blue_button_id
 			) h
-			JOIN suppressions s ON s.hicn = h.hicn and s.effective_date = h.max_date
-			WHERE preference_indicator = 'N'`).Pluck("hicn", &suppressedHICNs)
+			JOIN suppressions s ON s.blue_button_id = h.blue_button_id and s.effective_date = h.max_date
```

Comment:
We'll need to add an index on `blue_button_id` in the `cclf_beneficiaries` table.

Reply:
good catch!